Client/Frontend/Browser/BrowserViewController/BrowserViewController+ReaderMode.swift

```diff
@@ -0,0 +1,207 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
```

Comment:
Is this unchanged, you only moved it from the main BrowserViewController class?
Asking because i am not sure whether i should review all of this

Reply:
Nope while I was fixing the bug, I though this is great time to extract ReaderMode related protocols meanwhile.